hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/DeleteHoodieTableCommand.scala

```diff
@@ -65,7 +65,7 @@ case class DeleteHoodieTableCommand(deleteTable: DeleteFromTable) extends Hoodie
 
     assert(hoodieCatalogTable.primaryKeys.nonEmpty,
       s"There are no primary key defined in table $tableId, cannot execute delete operator")
-    withSparkConf(sparkSession, hoodieCatalogTable.catalogProperties) {
+    val globalConf = withSparkConf(sparkSession, hoodieCatalogTable.catalogProperties) {
```

Comment:
change to `val options: Map[String, String] = hoodieCatalogTable.catalogProperties.-(OPERATION.key)`?

Reply:
looks great